GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+
+When applicable, maintainers are listed in each file as the `__author__` string.
+
+### Core developers
+
+Core developers are contributors who have shown that they are dedicated to the continued development of the project through ongoing engagement with the community. They have shown they can be trusted to maintain sktime with care.
+
+Current core developers are listed in the [core-developers team](https://github.com/orgs/sktime/teams/core-developers/members) within the sktime organisation on GitHub.
+
+#### Rights and responsibilities
+
+| Right/responsibility | Description | 
+|---|---|
+| Direct access | Being a core developer allows contributors to more easily carry on with their project related activities by giving them direct access to the project’s repository. |
+| Issue/PR management | Core developers are responsible for reviewing and managing issues and pull requests. This includes commenting on issues, reviewing code contributions, merging approved pull requests, and closing issues once resolved. |
+| Decision making | They can partipate in the decision making process by vetoing changes and casting votes. |
+| Nomination | They can nominate new core developers, CoC committee members and CC members. | 
+
+#### Appointment
+New core developers can be nominated by any current core developer. Once they have been nominated, there will be a vote by the current core developers.
+
+Voting on new core developers is one of the few activities that takes place on the project's private chat. While it is expected that most votes will be unanimous, a 2/3 majority of the cast votes is enough. The vote needs to be open for at least one week.
+
+Core developers that have not contributed to the project (commits or GitHub comments) in the past 12 months will be asked if they want to become *emeritus core developers* and give up their rights and responsibilities until they become active again.
+
+### CoC committee members
+
+The current members of the CoC committee are listed in the [CoC](https://github.com/alan-turing-institute/sktime/blob/master/CODE_OF_CONDUCT.rst).
+
+#### Rights and responsibilities
+CoC committee members are responsible for investigating potential CoC incidents and enforcing the [CoC](https://github.com/alan-turing-institute/sktime/blob/master/CODE_OF_CONDUCT.rst). They are the point of contact for reporting potential CoC incidents. 
+
+In addition, they are responsible for maintaining and improving the CoC. 
+
+#### Appointment
+Nomination by core developers, vote by core developers, 2/3 majority and simple CoC majority
+
+To avoid deadlocks if there is an even number of CoC members, one of them will have a tie breaking priviledge. 
+
+### CC members
+CC members are core developers with additional rights and responsibilities to avoid deadlocks and ensure a smooth progress of the project. 
+
+The current members of the CC are:
```

Comment:
should lists like this be separate from the governance doc?
So the core governance doc is only changed when mechanisms/specs change (or typos/clarifications)